baybe/recommenders/pure/nonpredictive/clustering.py

```diff
@@ -91,7 +91,7 @@ def _make_selection_custom(
             candidates_scaled: The already scaled candidates.
 
         Returns:
-           A list with positional indices of the selected candidates.
+            A list with positional indices of the selected candidates.
```

Comment:
My editor claims that there was a change here, but I don't see any? Maybe some blanks/EOL stuff?

Reply:
There was one whitespace missing in the old code (i.e. not correctly indented) and I simply fixed it here